llvm/include/llvm/migration_cmd_examples.inc

```diff
@@ -0,0 +1,95 @@
+#ifdef _WIN32
```

Comment:
suggest to make is simple, like const char *longString = R""""(
This is 
a very 
long 
string
)"""";